ext/json/ext/generator/simd.h

```diff
@@ -0,0 +1,49 @@
+#include "extconf.h"
+
+typedef enum {
+    SIMD_NONE,
+    SIMD_NEON,
+} SIMD_Implementation;
+
+#ifdef ENABLE_SIMD
+
+#if defined(__ARM_NEON) || defined(__ARM_NEON__) || defined(__aarch64__) || defined(_M_ARM64)
+#include <arm_neon.h>
```

Comment:
The internet is convinced that no ARM64's lack neon support, so maybe this is not necessary. For example here: https://github.com/postgres/postgres/blob/REL_17_4/src/include/port/simd.h#L38